geomstats/geometry/lie_group.py

```diff
@@ -397,9 +397,9 @@ def log(self, point, base_point=None, point_type=None):
 
         return result
 
-    def exponential_barycenter(
-        self, points, weights=None, point_type=None
-    ):
+    def exponential_barycenter(self,
+                               points, weights=None,
```

Comment:
I thought we had the other indentation convention?

Reply:
Yes, good catch! I thought flake8 was throwing me an error there, but it must have been for something else. It is changed now, thanks!